# Hi there 👋

🎓 I'm studying for a master's degree in Bioinformatics at Queen Mary University of London.

**📚 Currently learning:**

Python for Bioinformatics (BioPython, sequence alignment, genome assembly), Machine Learning (PyTorch & Keras), Data Science (data preprocessing, data visualisation).

R for Statistical Analysis: hypothesis testing, linear models, data visualisation.

## Completed Projects:

[ML_Metabolomics_Crohns_Disease](https://github.com/WeronikaMazek/ML_Metabolomics_Crohns_Disease.git) ⬅︎
***Determining the Most Diagnostic Sample Type for Crohn's Disease Identification using GC-MS and Machine Learning***

## 🌱 Projects in Progress:

***Protein Secondary Structure Prediction***

*This project predicts protein secondary structures (alpha helices, beta sheets, and coils) from amino acid sequences using Keras. The work applies deep learning to sequence modeling, focusing on improving prediction accuracy and scalability for bioinformatics applications.*

・Key Features: Deep Learning for Sequence Modeling: Implementing RNNs and LSTMs in Keras to capture patterns in protein sequences.

・Scalable Solutions: Preprocessing and training on large datasets like CB513 to handle real-world bioinformatics challenges.

・Optimisation: Experimenting with hyperparameters, architecture tuning, and attention mechanisms for enhanced performance.

☑︎ *Current Progress:*

*Data preprocessing:* In Progress

*Model building and testing:* In Progress

*Optimisation and evaluation:* Upcoming

## 💻 Key skills & tools:

### Python Tools

Machine Learning: *Scikit-learn*

Data Science & Statistical Analysis: *Pandas, NumPy, Matplotlib, Seaborn, SciPy*

### R Tools

Visualisation: *ggplot2*

Data Manipulation: *dplyr, tidyr*

Bioinformatics: *Bioconductor, DESeq2*

## 📫 How to reach me:
📧 mazekweronika@gmail.com

🤝 I'm open to project collaborations and would appreciate any tips, advice, or guidance from the community. Feel free to reach out!
